scripts/replace-imports.sh

```diff
@@ -2,16 +2,18 @@
 
 set -euxo pipefail
 
-echo "replacing imports in $1"
+echo "Replacing imports from @trezor/*/src to @trezor/*/lib in folder $1"
 
 # Determine the operating system
 OS="$(uname)"
 
 # Execute the appropriate command based on the OS
 if [[ "$OS" == "Darwin" ]]; then
     # macOS command with -i '' for in-place editing without backup and -E for extended regex
-    find "$1" -type f -exec sed -i '' -E "s/import\(\"@trezor\/([^/]+)\/src/import(\"@trezor\/\1\/lib/g" {} +
+    # Removes the "import(" part from the search pattern and replaces @trezor/*/src with @trezor/*/lib
+    find "$1" -type f -exec sed -i '' -E "s/@trezor\/([^/]+)\/src/@trezor\/\1\/lib/g" {} +
```

Comment:
We don't want reference to `@trezor/*/src/` at all so replace it everywhere no matter if it's import/require or something else.